packages/docusaurus-theme-classic/src/theme/BlogSidebar/styles.module.css

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+.sidebar {
+  display: inherit;
+  max-height: calc(100vh - (var(--ifm-navbar-height) + 2rem));
+  overflow-y: auto;
+  position: sticky;
+  top: calc(var(--ifm-navbar-height) + 2rem);
+}
+
+.sidebarItemTitle {
+  margin-bottom: 0.5rem;
+}
+
+.sidebarItemList {
+  overflow-y: auto;
+  font-size: 0.8rem;
+  padding-left: 0.3rem;
```

Comment:
Maybe make the font a little bigger?

```suggestion
  font-size: .9rem;
  padding-left: 0;
```

| Before   | After    |
| -------- | -------- |
| ![image](https://user-images.githubusercontent.com/4408379/96284646-8c8ede00-0fe6-11eb-93d3-108912dca51c.png) | ![image](https://user-images.githubusercontent.com/4408379/96284533-5ea99980-0fe6-11eb-954c-83196007cbbe.png) |

Reply:
Why not. Ready to merge with your suggestion?